public/setup/atomicupdate/002_add_conges_index_twigization.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+$sql[] = "UPDATE `{$dbprefix}acces` SET `page`='/holiday' WHERE `page`='conges/index.php';";
+$sql[] = "UPDATE `{$dbprefix}menu` SET `url`='/holiday' WHERE `url`='conges/index.php';";
+$sql[] = "DELETE FROM `{$dbprefix}acces` WHERE `page` = '/holiday' AND `nom` <> 'Congés - Liste';";
```

Comment:
I think this 3 lines are useless.
updating conges/index.php to /holiday => We already have /holiday with the previous atomic update. So delete it (menu and acces)
So, the delete is no longer needed